.travis/yaml_splitter

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python3
+import yaml
+import sys
+import os
+
```

Comment:
A script header commentary could be beneficial. If someone were new and looking around, it would be some good context on why we have this script, and how the maps part of the website works.